commit b993502bebb9a0147b2f588756c07c810814b035 (refs/changes/77/4677/2)
Author: sjorge <sjorge@blackdot.be>
Date:   2018-08-15T17:35:19+02:00 (1 year, 2 months ago)
    
    OS-7116 bhyve brand should accept bios and uefi values for bootrom (fix
    cstyle)

diff --git a/usr/src/lib/brand/bhyve/zone/boot.c b/usr/src/lib/brand/bhyve/zone/boot.c
index 1e2c6771be..7c6b84f1ef 100644
--- a/usr/src/lib/brand/bhyve/zone/boot.c
+++ b/usr/src/lib/brand/bhyve/zone/boot.c
@@ -424,7 +424,7 @@ add_lpc(int *argc, char **argv)
 				val = DEFAULT_BOOTROM_CSM;
 			} else if (strcmp(val, "uefi") == 0) {
 				val = DEFAULT_BOOTROM;
-                        }
+			}
 		}
 		if (snprintf(conf, sizeof (conf), "%s,%s", lpcdevs[i], val) >=
 		    sizeof (conf)) {
